.gitignore

```diff
@@ -85,3 +85,6 @@ _releasenotes
 # Used during parts of the build process. Files _should_ get cleaned up automatically.
 # This is also a good location for any temporary manfiests used during development
```

Comment:
Please also bump the version in Tiltfile (+ tilt.md)

(just had to put the comment somewhere)